src/ios/CDVWKInAppBrowser.m

```diff
@@ -1175,6 +1175,18 @@ - (UIColor *)colorFromHexString:(NSString *)hexString {
 
 #pragma mark WKNavigationDelegate
 
+- (void)webView:(WKWebView *)theWebView didReceiveAuthenticationChallenge:(nonnull NSURLAuthenticationChallenge *)challenge completionHandler:(nonnull void (^)(NSURLSessionAuthChallengeDisposition, NSURLCredential * _Nullable))completionHandler {
+    NSString *domain = theWebView.URL.absoluteURL.host;
+    NSDictionary *credentials = _browserOptions.basicauth[domain];
```

Comment:
since we are using almost the same name below for the `NSURLCredential` object, can we rename this dictionary value to `login` (which is what I called it in android impl.) or something a bit more distinct?

Reply:
Good thing to be consistent indeed 👍